### PR TITLE
Activity Log: Filterbar add deselect and select all buttons

### PR DESCRIPTION
**Pull request by enejb**
Here we have the 3 new states, 
Initial view:
screen shot 2018-09-19 at 9 03 59 pm

screen shot 2018-09-19 at 9 04 08 pm

screen shot 2018-09-19 at 9 04 03 pm

Try this out and see if this makes sense. 
I didn't pay to much attention to the styling here. 

But the idea would be that it looks like the Date Range selector.

### CODE REVIEW

**Comment by enejb** [comment]
@keoshi I like your requested changes. I will wait for the decision on which way to go before I implement them. :)

**Comment by joanrho** [comment]
@enejb Let's keep the "select all" link and nix the word "checkboxes" for the "clear checkboxes" (just show "clear"). Thanks!

**Comment by joanrho** [comment]
@keoshi to answer your Q above, the "clear" should be one per popover. So one "clear" in the checkbox selector popover that clears out only that popover’s stuff and one "clear" in the date range selector popover that clears out only that popover’s stuff.

**Comment by keoshi** [comment]
@joanrho Got it, just asked for clarification because we already have a “clear dates” on the date selection:

image

And my suggestions were more to make them consistent but most importantly to capitalize the first word.

File: client/my-sites/activity/filterbar/date-range-selector.jsx [hunk 1]
```diff
@@ -278,7 +278,7 @@ export class DateRangeSelector extends Component {
 								{ from &&
 									to && (
 										<Button borderless compact onClick={ this.handleResetSelection }>
-											{ translate( '{{icon/}} clear dates', {
+											{ translate( '{{icon/}} clear', {
```
Comment (a8ci18n) [inline client/my-sites/activity/filterbar/date-range-selector.jsx]: :disappointed: 14 existing translations will be lost with this change.

**Comment by joanrho** [comment]
Let's make them both just "clear", since it's pretty obvious within each popover what the selection is for.

**Comment by enejb** [comment]
This works really well in my testing! Thanks for the changes @roccotripaldi

**Comment by keoshi** [comment]
Ditto, feels really smooth and works great!!

**Pull request merged**